docs/Playing/LT-Guide/mid-game.rst

```diff
@@ -0,0 +1,164 @@
+.. SPDX-License-Identifier: GPL-3.0-or-later
+.. SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+
+.. Custom Interpretive Text Roles for longturn.net/Freeciv21
+.. role:: unit
+.. role:: improvement
+.. role:: wonder
+.. role:: advance
+
+Turn 20+: Mid-Game Strategy
+***************************
+
+The Path to Democracy
+=====================
+
+While you are filling in the map with as many cities as you can, you are going to be developing your empire
+by terraforming tiles and adding infrastructure with :unit:`Workers` and at some point it will be necessary to
+add happiness buildings as well. Once you research :advance:`Monarchy` or :advance:`Republic` and take the two
+turns of Anarchy to change governments you will really see a change in output of all of your cities due to the
+elimination of the Despotism penalty.
+
+.. note::
+  As a reminder, you can also get rid of the Despotism penalty with the :wonder:`Pyramids` Small Wonder if
+  that fits your strategy better. However, at some point you **must** get to Democracy.
+
+You should to research the following technologies (not necessarily in this order):
```

Comment:
"You should to research..." - either the "to" is too much, or something is missing in this sentence.

Reply:
Thanks, good catch